### PR TITLE
Bump react-router's dependency to `^1.0.0-rc1`

### LINKED ISSUE #12: Update peerDependency of react-router
I tried the loader with the version 1.0.0-rc1 of react-router and it works fine. But npm gives trouble with the peerDependencies. It would be really nice to update this.

### PR DESCRIPTION
**Pull request by grrowl**
Fixes #12

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-router-proxy-loader",
-  "version": "0.3.0",
+  "version": "0.3.1",
```
Comment (seanadkinson) [inline package.json:3]: Lets go to `0.4.0` for `react-router@1.x`.  That will give us space to have `0.3.1` for bug fixes on `react-router@0.x`

**Comment by seanadkinson** [comment]
I'll merge and update this.

**Comment by seanadkinson** [comment]
Pushed version `0.4.0` to NPM

File: package.json (hunk 1 above)
Comment (grrowl) [inline package.json:3]: You're totally right, plus it does affect behaviour external to the project so it should be a minor at least.

**Pull request merged**